### ISSUE TITLE
Provide "you are logged in" UI element

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-auth-github#5]
Can the plugin add this to the template without needing to first modify Datasette core?

**Comment by simonw** [comment]
No obvious hooks in https://github.com/simonw/datasette/blob/master/datasette/templates/base.html - also the Datasette design doesn't actually have any universal navigation that would be a good fit for a "you are logged in as" widget. Not obvious how to proceed with this one.

**Comment by simonw** [comment]
For the moment I could put this in the footer? That's not easy to customize right now - it's hard-coded into the `base.html` template - but I could modify Datasette core to turn that into an overridable block (or an include, or both) to make this easier to hook into.

**Comment by simonw** [comment]
If I'm going to need to modify the core templates for this anyway, it's probably time to introduce a universal navigation bar at the top of the page, rather than mess around too much with the footer.

It's fine to couple this to a new Datasette release because this plugin already uses a new plugin hook which will need to be released with Datasette 0.29.

**Comment by simonw** [comment]
Here's a `base.html` template that takes advantage of the new code in https://github.com/simonw/datasette/issues/540
```html+django
{% extends "default:base.html" %}

{% block extra_head %}
<style type="text/css">
.hd .logout {
    float: right;
    text-align: right;
    padding-left: 1em;
}
</style>
{% endblock %}

{% block nav %}
    {{ super() }}
    <p class="logout">
        <a href="/-/logout">Log out</a>
    </p>
{% endblock %}
```
